scripts/migrations/2020-10-miro-migration/transfer_packager.py

```diff
@@ -0,0 +1,166 @@
+import concurrent
+import itertools
+import os
+import shutil
+
+from tqdm import tqdm
+
+S3_DOWNLOAD_CONCURRENCY = 3
+
+
+def _assert_local_content_length(file_location, expected_content_length):
+    # Assert we have the correct number of bytes
+    local_content_length = os.path.getsize(file_location)
+
+    # Check that content length is non-zero
+    assert local_content_length > 0
+
+    assert (
+        local_content_length == expected_content_length
+    ), f"Content length mismatch: {file_location}!"
```

Comment:
Notes:

* I would call this `check` rather than `assert`. It does an assertion, but the purpose of this function is to check the content length matches than in S3.
* The first comment should be the docstring, not a comment.
* I would log the expected/actual content lengths in the assertion.